forc-plugins/forc-migrate/src/visiting/mod.rs

```diff
@@ -634,6 +634,9 @@ impl __ProgramVisitor {
                     Self::visit_expr(ctx, lexed_returned.__as_ref(), ty_returned, visitor, output)?;
                 }
             }
+            Expr::Panic { expr_opt: _, .. } => {
+                // TODO: Implement visiting `panic`.
```

Comment:
Would be nice to keep track of this.

Reply:
For `forc migrate` tool, there a lot of expressions that are not covered. They are all marked as TODO. The plan is not to invest the effort upfront into covering them but rather as a part of writing concrete migrations, if a need is there. That's why I didn't see it as necessary to open an issue for it. It is a known ongoing effort.